### PR TITLE
First attempt to fix #3993. 

### LINKED ISSUE #3993: Some images don't load on macOS Safari nor in any iOS browsers on lbry.tv
**Bug**
The image at https://lbry.tv/@LBRY-Digest:a/Ever-the-optimist-April-Fools-Day:3 does not load in Safari on macOS on lbry.tv. It also does not load in Safari, Firefox, or Chrome on iOS on lbry.tv.

**To Reproduce**
1. In Safari on macOS or on any browser in iOS, go to https://lbry.tv/@LBRY-Digest:a/Ever-the-optimist-April-Fools-Day:3

**Expected behavior**
The image should load and be displayed

**Screenshots**
Screen Shot 2020-04-14 at 11 32 01 AM

**System Configuration**

- LBRY Daemon version:  0.68.0
- LBRY App version:
- LBRY Installation ID: 692EAWhtoqDuAfQ6KHMXxFxt8tkhmt7sfprEMHWKjy5hf6PwZcHDV542VHqRnFnTCD
- Operating system: macOS Catalina, Version 10.15.3 and iOS version 13.3.1

**Additional context**

### PR DESCRIPTION
**Pull request by jeffslofish**
Show error message if image is not able to be displayed by browser.

## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #3993

## What is the current behavior?
If a browser cannot display an image (such as Safari and WebP images), then an empty box is displayed in place of the image.

## What is the new behavior?
If a browser cannot display an image, an error message is displayed in a Card.

### CODE REVIEW

**Comment by seanyesmunt** [comment]
Thanks for this PR @jeffslofish! I'll review this today!

**Comment by tzarebczan** [comment]
@jeffslofish Thanks again for the contribution! Reach out to us so we can [show you some appreciation](https://lbry.com/faq/appreciaiton).

File: ui/component/viewers/imageViewer.jsx [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+
   return (
-    <div className="file-render__viewer">
-      <img src={source} />
-    </div>
+    <React.Fragment>
+      {loadingError && (
+        <Card
+          title={__('Error Displaying Image')}
+          defaultExpand
```
Comment (seanyesmunt) [inline ui/component/viewers/imageViewer.jsx:19]: Can you remove this? That's only used for cards that should be collapsable.

**Comment by jeffslofish** [comment]
> Thanks for your contribution @jeffslofish 🙌
> 
> It's pretty obvious you know what you're doing, so if you have any thoughts or feedback on the setup and introductory development experience, we'd welcome your feedback.

@kauffj Yes I do have some feedback on the setup and introductory developer experience. Where is a good place to leave that feedback?

**Comment by seanyesmunt** [comment]
@jeffslofish I just created this issue. Can you comment there?
https://github.com/lbryio/lbry-desktop/issues/4036

**Pull request merged**